draft-ietf-quic-transport.md

```diff
@@ -1541,112 +1731,6 @@ number. "packet_number_secret" is derived from the TLS key exchange,
 as described in Section 5.6 of {{QUIC-TLS}}.
```

Comment:
The packet-number-gap trick only works if both the client and the server do it, and if both refrain from sending any packet on the old connection in parallel with the new connection. That seems very fragile. We should really consider encrypting the packet sequence number.

Note that all this discussion only applies to voluntary migration. It does not apply to NAT rebinding. If the client cannot anticipate the NAT rebinding, it will not be able to switch to a new connection ID or to insert a gap in the sequence numbers.

Reply:
See issue #1034. The packet number gap is a complex mechanism that does not actually protect privacy, because client and server apply the same gap.

Reply:
I agree with Christian, this seems fragile, but that's for another issue and discussion